portal/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.server.Directives.{ complete, get, onComplete, parameter, path }
+
+import scala.util.Failure
+import cromwell.pipeline.service.UserService
+import akka.http.scaladsl.model.{ StatusCodes }
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.Route
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport
+
+import scala.concurrent.ExecutionContext
+import scala.util.Success
+
+class UserController(userService: UserService)(implicit executionContext: ExecutionContext) extends PlayJsonSupport {
+
+  val route: Route =
+    path("users") {
+      get {
+        parameter('email.as[String]) { email =>
+          onComplete(userService.getUsersByEmail(email)) {
+            case Success(r) => complete(r)
+            case Failure(_) =>
+              complete(
+                StatusCodes.InternalServerError,
+                "may be, database error or query syntax error: try to check your request email-pattern"
+              ) //TODO - add text description for this error
```

Comment:
Remove all comments in the PR if we don't need to see them.

Reply:
removed